collector/app/utils/settings.py

```diff
@@ -0,0 +1,24 @@
+"""This module provides utils for parsing config."""
+
+import argparse
+import os
+
+DEFAULT_HOST = os.getenv('DEFAULT_HOST')
+DEFAULT_PORT = os.getenv('DEFAULT_PORT')
+
+
+def get_config(argv=None):
```

Comment:
Since we decided to use .env configuration in all parts of the project: `default_host` and `default_port` are default, then you just need to read it from environ as default parameter (os.eviron.get("COLLECTOR_HOST", "localhost").
.env file is created for the case when a developer wants to set up his own hostname and port, in the other way he'll use default hostname and port which are public to know.

Therefore, what I expect to see. There are a few lines of code:
```
def load_configs():
    return {
        "host": os.environ.get("COLLECTOR_HOST", "localhost"),
        "port": os.environ.get("COLLECTOR_PORT", 5000)
    }
```

Or, if you want to leave ability to set host and port via command arguments: just add default paramater to the `add_argument` command and remove part where default values are loading from the .env file.

```
def load_configs(argv)
    parser = argparse.ArgumentParser()
    parser.add_argument("--host", type=str, default="localhost",
                        help="Setup application host")
    parser.add_argument("--port", type=int, default=5000,
                        help="Setup application port")

    # ignore unknown options
    options, _ = parser.parse_known_args(argv)
    config = {
        'host': options.host,
        'port': options.port
    }
    return config
```